src/store/models/wallet.ts

```diff
@@ -4,10 +4,32 @@ import type { RootModel } from './models';
 
 export interface WalletState {
   chosenAsset?: string;
+  assets: Array<{
+    name: string;
+    ticker: string;
+    amount: number;
+    price: number;
+  }>;
 }
 
 export default createModel<RootModel>()({
-  state: { chosenAsset: 'Total Holdings' } as WalletState,
+  state: {
+    chosenAsset: 'Total Holdings',
```

Comment:
will populate this from the response from the chain in further PRs, or maybe will change it up 🤷🏽  Havent decided yet.

Reply:
Could potentially also have it so that when `chosenAsset` is `null` is when we show the "Total Holdings" view, rather than relying on a string comparison.